### ISSUE TITLE
Remove react-addons-css-transition-group from package.json

### ISSUE THREAD

**Issue by BerkeleyTrue** [freeCodeCamp/freeCodeCamp#14476]
We are no longer using this module

**Comment by Masya120** [comment]
I'd like to take this one

**Comment by BerkeleyTrue** [comment]
@Masya120 Cool. Please read https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md before starting if you haven't already done so.

Happy coding

**Comment by BerkeleyTrue** [comment]
@Masya120 My apologies. It looks like this two packages where added to solve an issue with React and react-hot-loader. This package needs to stay until we can update to the latest version of react-hot-loader, which is still in beta, and react/react-dom.
